lib/inquisition/rails_best_practices/runner.rb

```diff
@@ -3,14 +3,23 @@
 module Inquisition
   module RailsBestPractices
     class Runner < ::Inquisition::Runner
+      USER_CONFIG_PATH = 'rails_best_practices.yml'.freeze
```

Comment:
From the readme of the gem `configuration file location (defaults to config/rails_best_practices.yml)`